test/Driver/emit-sil-single-file.swift

```diff
@@ -12,14 +12,6 @@
 // CHECK: Hello World
 // CHECK: Hello Bob, today is Tuesday.
 
-// This test intentionally mirrors /Driver/emit-sib-single-file to ensure that
-// SwiftOnoneSupport is always a dependency of -Onone -emit-si*gen builds.
```

Comment:
You can either collapse this and its SIBGen sibling together and move them with other executable tests or straight-up remove them after you’re done here.

Reply:
Yeah I was hoping to just remove them in The Real Patch™